src/interfaces.ts

```diff
@@ -0,0 +1,14 @@
+import { FastifyServerOptions } from "fastify";
+
+export interface config {
```

Comment:
what's the rationale on using `interface` vs `type`?  do we expect user to `implement` config?

Reply:
I don't know typescript. Moved to export a type from this module instead of an interface. I was just following what I had seen in the wild for configuration options, but moved to types to intersect interfaces.